wercker.yml

```diff
@@ -17,12 +20,4 @@ build:
             code: |
                 export RETHINKDB_HOST=$WERCKER_RETHINKDB_HOST
                 tox
-deploy:
```

Comment:
I was a bit wary leaving it in there for wercker while I was learning it, didnt know if it would actually deploy, and given development branch, just thought it best to take it out.